src/Symfony/Component/VarDumper/Tests/Fixtures/Twig.php

```diff
@@ -26,14 +26,11 @@ public function getTemplateName()
 
     public function getDebugInfo()
     {
-        return array (19 => 2);
+        return array(19 => 2);
     }
 
-    public function getSource()
+    public function getSourceContext()
     {
-        return "   foo bar
-     twig source
-
-";
+        return new Twig_Source("   foo bar\n     twig source\n\n", 'foo.twig', 'foo.twig');
```

Comment:
Bof, that's only a caster, adding a test for that would be quite a bit of work for a minor thing to me...